esrally/driver/runner.py

```diff
@@ -310,21 +310,24 @@ async def __call__(self, es, params):
 
         * ``body``: containing all documents for the current bulk request.
         * ``bulk-size``: the number of documents in this bulk.
-        * ``action_metadata_present``: if ``True``, assume that an action and metadata line is present (meaning only half of the lines
-        contain actual documents to index)
+        * ``action_metadata_present``: if ``True``, assume that an action and metadata line is present (meaning only
+                                       half of the lines contain actual documents to index)
         * ``index``: The name of the affected index in case ``action_metadata_present`` is ``False``.
         * ``type``: The name of the affected type in case ``action_metadata_present`` is ``False``.
 
         The following keys are optional:
 
         * ``pipeline``: If present, runs the the specified ingest pipeline for this bulk.
-        * ``detailed-results``: If ``True``, the runner will analyze the response and add detailed meta-data. Defaults to ``False``. Note
-        that this has a very significant impact on performance and will very likely cause a bottleneck in the benchmark driver so please
-        be very cautious enabling this feature. Our own measurements have shown a median overhead of several thousand times (execution time
-         is in the single digit microsecond range when this feature is disabled and in the single digit millisecond range when this feature
-         is enabled; numbers based on a bulk size of 500 elements and no errors). For details please refer to the respective benchmarks
-         in ``benchmarks/driver``.
-
+        * ``detailed-results``: If ``True``, the runner will analyze the response and add detailed meta-data. Defaults
+                                to ``False``. Note that this has a very significant impact on performance and will very
+                                likely cause a bottleneck in the benchmark driver so please be very cautious enabling
+                                this feature. Our own measurements have shown a median overhead of several thousand
+                                times (execution time is in the single digit microsecond range when this feature is
+                                disabled and in the single digit millisecond range when this feature is enabled; numbers
+                                based on a bulk size of 500 elements and no errors). For details please refer to the
+                                respective benchmarks in ``benchmarks/driver``.
+        * ``request-timeout``: a non-negative float indicating the client-side timeout for the operation.  If not
+                               present, defaults to ``None`` and potentially falls back to the global timeout setting.
```

Comment:
Please apply the same line length formatting as the other multi line length comments